test/unit/int_algolia/scripts/algolia/steps/__snapshots__/algoliaProductDeltaIndex.test.js.snap

```diff
@@ -239,427 +239,4 @@ exports[`process default 1`] = `
 ]
 `;
 
-exports[`process master-level indexing 1`] = `
```

Comment:
Such a big diff in tests result should draw your attention.
Basically the function was previously generating records and now return an empty array.
You have to understand why before updating the test to make it pass. Here, you are potentially hiding a bug!

The problem can just come from the mock, in which case you can update it, but from what I see, your implementation of `isIncludeOutOfStock` differs from what you had initially done in `algoliaLocalizedProduct.js`. Is it normal?